doc/info/pdp-10.16

```diff
@@ -835,16 +835,16 @@ DIVB	C(AC) <- C(AC AC+1) / C(E); C(AC+1) <- remainder;
 
 PDP-10  Node: KL-Only, Previous: Fixed Point, Up: Top, Next: Floating Point
 
-This node describes some instructions that only KL-10's have.  This
-means that the only ITS machine which has them is MC.  You should
-probably not use them even if you expect to run on MC, so that your
-program can be moved.
+This node describes some instructions that only KL-10's and KS10's
```

Comment:
This is incorrect English grammar.  The apostrophe signifies the possessive case.  It says that the KL-10s and KS10s have something.  Making something plural, as here, requires no apostrophe, but just the "s".  These should be KL-10s and KS10s.